test/unit/CodeGenerator/Definitions/RequestBodyDtoDefinitionTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\CodeGenerator\Definitions;
+
+use OnMoon\OpenApiServerBundle\CodeGenerator\Definitions\PropertyDefinition;
+use OnMoon\OpenApiServerBundle\CodeGenerator\Definitions\RequestBodyDtoDefinition;
+use OnMoon\OpenApiServerBundle\Specification\Definitions\Property;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \OnMoon\OpenApiServerBundle\CodeGenerator\Definitions\RequestBodyDtoDefinition
+ */
+final class RequestBodyDtoDefinitionTest extends TestCase
```

Comment:
This test is not needed, RequestBodyDtoDefinition is an empty class and DtoDefinition has its own test.